collectors/o365/o365_collector.js

```diff
@@ -129,7 +125,7 @@ class O365Collector extends PawsCollector {
 
             if(nextPageUri && pageCount < process.env.paws_max_pages_per_invocation){
                 pageCount++;
-                return m_o365mgmnt.getPreFormedUrl(collector.pawsCred, nextPageUri)
+                return m_o365mgmnt.getPreFormedUrl(collector.o365_mgmnt_client, nextPageUri)
```

Comment:
I think we can call it like `collector.o365_mgmnt_client. getPreFormedUrl(nextPageUri)`